src/clj/game/cards/operations.clj

```diff
@@ -1191,6 +1191,29 @@
     :msg "shuffle a card from HQ into R&D"
     :effect (final-effect (move target :deck) (shuffle! :deck))}
 
+   "Riot Suppression"
+   {:req (req (last-turn? state :runner :trashed-card))
+    :delayed-completion true
+    :effect (req (show-wait-prompt state :corp "Runner to decide if they will take 1 brain damage")
+                 (when-completed (resolve-ability
+                                   state :runner
+                                   {:optional
+                                    {:prompt "Take 1 brain damage to prevent having 3 fewer clicks next turn?"
+                                     :player :runner
+                                     :delayed-completion true
```

Comment:
Remember, the optional itself is never delayed-completion, only its yes/no-ability might be.